resources/hg-ReferralComposition.xml

```diff
@@ -1,534 +1,665 @@
+        <valueString value="Zorgpad" />
+      </example>
+    </element>
+    <element id="Composition.section:envelope.section:carePath.code">
+      <path value="Composition.section.section.code" />
+      <min value="1" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="370858005" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Composition.section:envelope.section:requiredConsultationFacilities">
+      <path value="Composition.section.section" />
+      <sliceName value="requiredConsultationFacilities" />
+      <max value="1" />
+    </element>
+    <element id="Composition.section:envelope.section:requiredConsultationFacilities.extension">
+      <path value="Composition.section.section.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="1" />
+    </element>
+    <element id="Composition.section:envelope.section:requiredConsultationFacilities.extension:requiredConsultationFacilitiesTextValue">
+      <path value="Composition.section.section.extension" />
+      <sliceName value="requiredConsultationFacilitiesTextValue" />
+      <min value="1" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-TextValue" />
+      </type>
+    </element>
+    <element id="Composition.section:envelope.section:requiredConsultationFacilities.extension:requiredConsultationFacilitiesTextValue.value[x]">
+      <path value="Composition.section.section.extension.value[x]" />
+      <short value="RequiredConsultationFacilities" />
+      <definition value="Geeft de extra maatregelen of aandachtspunten die nodig zijn voor een consult met deze patiënt.&#xD;&#xA;&#xD;&#xA;De HASP-richtlijn (NHG) geeft als voorbeeld een tolkentelefoon." />
+      <alias value="VoorzieningenNodigBijConsult" />
+      <min value="1" />
+      <mapping>
+        <identity value="ART-DECOR-hg-Envelop" />
+        <map value="hg-dataelement-1708" />
+        <comment value="VoorzieningenNodigBijConsult" />
+      </mapping>
+    </element>
+    <element id="Composition.section:envelope.section:requiredConsultationFacilities.title">
+      <path value="Composition.section.section.title" />
+      <example>
+        <label value="Example for referral" />
+        <valueString value="Voorzieningen nodig bij consult" />
+      </example>
+    </element>
+    <element id="Composition.section:envelope.section:requiredConsultationFacilities.code">
+      <path value="Composition.section.section.code" />
+      <min value="1" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="230211000146109" />

```

Comment:
The update of this code is not explicitly mentioned in ELZT-80 or ELZT-84. Is this tracked in another ticket, or should we add it in the release notes for traceability?

Reply:
Fair point, will add this.